environments/demo/demo-platform-hmcts-net.yml

```
@@ -29,7 +29,7 @@ A:
     ttl: 300
   - name: sds-api-mgmt
     record:
-    - 10.51.97.133
+    - 10.51.99.4
```

```suggestion
    - 10.51.97.xx
```
Need to wait until sds apim demo is back into original subnet before we can update the dns